app/views/home/index.html.erb

```diff
@@ -0,0 +1,57 @@
+    <nav class="navbar navbar-default navbar-expand-lg fixed-top">
+    <div class="container flex justify-content0-between align-items-center">
+
+      <span></span>
+     
+      <h6 class="title">CATEGORIES</h6>
+
+
+      <div></div>
+
+
+    </div>
+  </nav>
+<div class="d-flex justify-content-between align-items-center bg-white p-2">
+  <h6 class="pt-2">Categories</h6>
+
+  <%= link_to 'Add Category', new_group_path, class: 'btn btn-primary' %>
+</div>
+
+<div class="mb-5">
+
+<% if @groups.empty? %>
+
+    <div class="d-flex flex-column justify-content-center align-items-center m-5">
+
+        <p>Category is empty.</p>
+        <%= link_to 'Add Category', new_group_path, class: 'btn btn-primary' %>
+
+     </div>
+<% else %>
+
+
+ <% @groups.each do |group|%>
+     <%= link_to controller: 'groups', action: 'show', id: group.id do %>
+          <div class="d-flex justify-content-between align-items-center bg-white p-2 gap-6 m-2 shadow-sm" >
+
+            <div class="d-flex justify-content-start align-items-center gap-3">
+              <img src="<%= group.icon %>" height="100" width="100">
+              <div class="flex-column d-flex justify-content-between mt-4">
+                <h6 class="pb-3 text-black-60"><%= group.name.capitalize %></h6>
+                <p class="text-black-50"><%= group.created_at.strftime("%Y-%m-%d") %></p>
+              </div>
+            </div>
+            <div class="fw-bold">$<%= group.entities.sum(:amount) %></div>
+        </div>
+      <% end %>
+    <% end %>
+<% end %>
+
+</div>
+  <div class="position-fixed bottom-0 w-100">
+    <div class="d-flex justify-content-center align-items-center bg-white p-2">
+
+  <%= link_to 'Add Category', new_group_path, class: 'btn btn-primary w-full w-50' %>
+</div>
+</div>
```

Comment:
- [ ] As mentioned in the project requirements you should follow the given design "Colors, Typography: font face, size and weight, Layout: composition and space between elements"

Your design

<img width="371" alt="Screenshot 2023-10-21 at 18 28 12" src="https://github.com/yemidada/MyBudget/assets/74501523/e3ef11ab-9e4c-4fbf-abaf-79113fd0cff7">

Given design

<img width="402" alt="Screenshot 2023-10-21 at 19 12 03" src="https://github.com/yemidada/MyBudget/assets/74501523/42ab5339-dbee-4e22-96d4-bec12a08edc4">

- [ ] Also, you have many `Add Category` buttons on this page, it would be nice to have only one button at the bottom as shown in the given design 😁 😁